deps/zlib/zconf.h

```diff
@@ -8,6 +8,18 @@
 #ifndef ZCONF_H
 #define ZCONF_H
 
+/*
+ * This library is also built as a part of AOSP, which does not need to include
+ * chromeconf.h. This config does not want chromeconf.h, so it can set this
+ * macro to opt out. While this works today, there's no guarantee that building
+ * zlib outside of Chromium keeps working in the future.
+ */
+//#if !defined(CHROMIUM_ZLIB_NO_CHROMECONF)
+/* This include does prefixing as below, but with an updated set of names.  Also
+ * sets up export macros in component builds. */
+//#include "chromeconf.h"
+//#endif
```

Comment:
Any particular reason this is commented out instead of `#define`ing `CHROMIUM_ZLIB_NO_CHROMECONF` in `zlib.gyp`?

Reply:
IIRC there were issues with zlib addon or similar tests.

Reply:
The define would have to end up in `common.gypi`, I think.

Reply:
I can do that if that's more desirable.